instructions/FOSS_staging_list.pl

```diff
@@ -20,11 +20,9 @@
       "deploy_pkg_into" => "zimbra-zextras",
    },
    {
-      "dir"         => "zm-timezones",
-      "ant_targets" => ["publish-local"],
-      "stage_cmd"   => sub {
-         System("(cd .. && rsync -az --relative zm-timezones $CFG{BUILD_DIR}/)");
-      },
+      "dir"             => "zm-timezones",
+      "ant_targets"     => ["pkg"],
+      "deploy_pkg_into" => "zimbra-foss",
```

Comment:
I'm unsure what this means.  Will zm-timezones be in its own package - which would be useful for independant updating or will it be part of a zimbra-foss package which has lots of other baggage?